instrumentation/instabeego/README.md

```diff
@@ -0,0 +1,65 @@
+Instana instrumentation for Beego framework
+=============================================
+
+This module contains middleware to instrument HTTP services written with [`https://github.com/beego/beego`](https://github.com/beego/beego).
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/instana/go-sensor/instrumentation/instabeego)][godoc]
+
+Installation
+------------
+
+To add the module to your `go.mod` file run the following command in your project directory:
+
+```bash
+$ go get github.com/instana/go-sensor/instrumentation/instabeego
+```
+
+Usage
+-----
+
+## Web server instrumentation
+
+```go
+// create a sensor
+sensor := instana.NewSensor("beego-server")
```

Comment:
@Angith :
Would it be preferable to use `InitCollector` instead of `NewSensor`? In any case, we require an instance of `TracerLogger` interface for instrumentation, right?

The term `sensor` might imply a different concept, doesn't it? How about using `collector` or `tracer` in our new examples for clarity?

Reply:
good catch! done